src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -226,7 +229,10 @@ bool AclEltwiseExecutor::init(const EltwiseAttrs &eltwiseAttrs, const std::vecto
                 return false;
             exec_func = [this]{
                 auto acl_op = std::make_unique<NEElementwiseDivision>();
-                acl_op->configure(&srcTensors[0], &srcTensors[1], &dstTensors[0]);
+                {
+                    std::unique_lock<std::mutex> lock(aclArithmeticLock);
+                    acl_op->configure(&srcTensors[0], &srcTensors[1], &dstTensors[0]);
+                }
```

Comment:
should we report the issue on ACL team? Looks strange that we configure specific op and it leads to segfault